skdownscale/pointwise_models/core.py

```diff
@@ -306,6 +306,8 @@ def _to_feature_x(self, X, feature_dim=DEFAULT_FEATURE_DIM):
         if feature_dim not in X.dims:
             X = X.expand_dims(**{feature_dim: [f'{feature_dim}_0']}, axis=1)
 
+        # all features must be in 1 chunk for map_blocks to work later on
+        X = X.chunk({feature_dim: -1})
```

Comment:
this looks good except I think it bakes in a dask dependency here. Instead, I think we need to check to see if the dataset is already chunked:

```suggestion
        if X.chunks:
            X = X.chunk({feature_dim: -1})
```